operator/build/Dockerfile

```diff
@@ -25,6 +25,10 @@ LABEL name="IBM Spectrum Scale CSI Operator" \
       description="An Ansible based operator to run and manage the deployment of the IBM Spectrum Scale CSI Driver." \
       git_commit=$GIT_COMMIT
 
+#COPY requirements.yml ${HOME}/requirements.yml
```

Comment:
should we just delete these comments?

Reply:
Probably, I'll nix them in the merge commit.